pkg/kubectl/cmd/get.go

```diff
@@ -133,6 +133,7 @@ func NewCmdGet(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.Comman
 	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 	cmdutil.AddIncludeUninitializedFlag(cmd)
 	cmd.Flags().BoolP("watch", "w", false, "After listing/getting the requested object, watch for changes. Uninitialized objects are excluded if no object name is provided.")
+	cmd.Flags().String("field-selector", "", "Selector (field query) to filter on, supports '=', '==', and '!='.(e.g. --field-selector key1=value1,key2=value2)")
```

Comment:
Add a second sentence to the field description that `The server only supports a limited number of field queries per type.`

Reply:
ping @smarterclayton Updated. PTAL. Thanks.